packages/EasyCore/src/Bridge/Symfony/Traits/InvalidDataGeneratorTrait.php

```diff
@@ -0,0 +1,535 @@
+        $fewerArrayCollection = new ArrayCollection($fewerElements);
+
+        return [
+            "${property} has too few elements in the collection" => [
+                'data' => [
+                    $property => $fewerArrayCollection,
+                ],
+                'message' => "This collection should contain ${minElements} element or more.",
+            ],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getArrayWithFewerItems(string $property, int $minElements): array
+    {
+        return [
+            "${property} has too few elements in the array" => [
+                'data' => [
+                    $property => \array_fill(0, $minElements - 1, null),
+                ],
+                'message' => "This collection should contain ${minElements} element or more.",
+            ],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getArrayWithMoreItems(string $property, int $maxElements): array
+    {
+        return [
+            "${property} has too many elements in the collection" => [
+                'data' => [
+                    $property => \array_fill(0, $maxElements + 1, []),
+                ],
+                'message' => "This collection should contain ${maxElements} elements or less.",
+            ],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getBlankString(string $property): array
+    {
+        return [
+            "${property} is blank" => [
+                'data' => [
+                    $property => '',
+                ],
+                'message' => 'This value should not be blank.',
+            ],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getBlankStringInArray(string $property): array
+    {
+        return [
+            "${property} is blank" => [
+                'data' => [
+                    $property => [''],
+                ],
+                'message' => 'This value should not be blank.',
+            ],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getDateTimeLessThanOrEqualToNow(string $property): array
+    {
+        $dateTime = Carbon::now();
+
+        return [
+            "${property} has less datetime" => [
+                'data' => [
+                    $property => $dateTime->clone()
+                        ->subSecond()
+                        ->toAtomString(),
+                ],
+                'message' => 'This value should be greater than now.',
+            ],
+            "${property} has equal datetime" => [
+                'data' => [
+                    $property => $dateTime->toAtomString(),
+                ],
+                'message' => 'This value should be greater than now.',
+            ],
+        ];
+    }
+
+    /**
+     * @return mixed[]
+     */
+    public function getEmptyArrayCollection(string $property, int $minElements): array
```

Comment:
Can we call `getArrayCollectionWithFewerItems` inside of this method to not repeat the logic?